### PR TITLE
Download plugins

### PR DESCRIPTION
**Pull request by zolotov**

### CODE REVIEW

File: cli/src/main/kotlin/client.kt [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+class Client {
+    companion object {
+        @JvmStatic
+        fun main(args: Array<String>) {
+            if (args.isEmpty()) {
+                System.err.println("Command is not specified: `upload` or `download` commands are supported.")
+                System.exit(1)
+            }
+            val command = args[0]
+            val restParameters = if (args.size > 1) args.copyOfRange(1, args.size) else emptyArray()
```
Comment (chashnikov) [inline cli/src/main/kotlin/client.kt]: Why do you need this? Doesn't 'copyOfRange(1,1)' return empty array anyway?

**Comment by zolotov** [comment]
Updated and squashed

**Comment by zolotov** [comment]
Why did you merge this? I was going to fix all review-notes as a part of this pull-request

**Comment by chashnikov** [comment]
Because I only found minor things, and I supposed it was important for you to have these changes merged soon.

**Comment by zolotov** [comment]
Ok, thank you

**Pull request merged**